### ISSUE TITLE
(intellect.py, line 205)   invalid syntax

### ISSUE THREAD

**Issue by mgrouch** [pypilot/pypilot#82]
ERROR loading intellect.py invalid syntax (intellect.py, line 205)   invalid syntax (intellect.py, line 205)
ERROR loading learning.py invalid syntax (intellect.py, line 205)   invalid syntax (intellect.py, line 205)

**Comment by mgrouch** [comment]
self.model.compile(optimizer='adam', loss='mean_squared_error', metrics=['mse'])
        self.accuracy =


self.accurracy = 

is not set

**Comment by seandepagnier** [comment]
just ignore this warning, the learning pilot doesn't work yet anyway
